astropy/io/misc/asdf/tags/transform/compound.py

```diff
@@ -11,26 +11,26 @@
 
 
 _operator_to_tag_mapping = {
-    '+'  : 'add',
-    '-'  : 'subtract',
-    '*'  : 'multiply',
-    '/'  : 'divide',
-    '**' : 'power',
-    '|'  : 'compose',
-    '&'  : 'concatenate',
+    '+': 'add',
+    '-': 'subtract',
+    '*': 'multiply',
+    '/': 'divide',
+    '**': 'power',
+    '|': 'compose',
+    '&': 'concatenate',
     'fix_inputs': 'fix_inputs'
 }
 
 
 _tag_to_method_mapping = {
-    'add'         : '__add__',
-    'subtract'    : '__sub__',
-    'multiply'    : '__mul__',
-    'divide'      : '__truediv__',
-    'power'       : '__pow__',
-    'compose'     : '__or__',
-    'concatenate' : '__and__',
-    'fix_inputs'  : 'fix_inputs'
+    'add': '__add__',
+    'subtract': '__sub__',
+    'multiply': '__mul__',
+    'divide': '__truediv__',
+    'power': '__pow__',
+    'compose': '__or__',
+    'concatenate': '__and__',
+    'fix_inputs': 'fix_inputs'
```

Comment:
I find this more difficult to read but whatever ...

Reply:
Do you want me to revert this? I was just following the `flake8` suggestions.

Reply:
Not that it's terrible this way, it's just easier to read so if you don't mind.  Thanks for cleaning up.

Reply:
What about now? Turns out `flake8` doesn't care for spacing after the `:` but not before.

Reply:
it's good now 👍